packages/strapi-utils/lib/__tests__/sanitizeEntity.test.js

```diff
@@ -0,0 +1,217 @@
+const sanitizeEntity = require('../sanitize-entity');
```

Comment:
You can merge this with the new tests that have been written, also we're trying to move from `camelCase` to `kebab-case` for filenames.